src/heuristics/Doc2Vec.py

```diff
@@ -19,7 +20,8 @@ def __init__(self, is_cleaned):
 
     def setup(self, api, start, goal):
         self.model = Doc2Vec.load(
-            "models/" + ("doc2vec_cleaned" if self.is_cleaned else "doc2vec"))
+            os.path.abspath(__file__)[:-len("src\\heuristics\\Doc2Vec.py")] + "models\\" +
```

Comment:
Won't the backslashes only work on Windows?

Reply:
Oh idk thats what it printed out when i debugged it. Should I run through it and replace every "\\" with "/"?

Reply:
There must be a builtin way to handle paths like this; string replacement seems disgusting

Reply:
hmmmmmmmmmmmmmmmmm